src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -308,3 +310,96 @@ export const answerReviewLabel = (key, formValues) => {
     }
   }
 };
+
+export const determineBoardObj = (formResponses, noDRB) => {
+  if (!formResponses) return null;
+
+  const prevAppType = [
+    RESPONSES.PREV_APPLICATION_TYPE_1,
+    RESPONSES.PREV_APPLICATION_TYPE_4,
+  ].includes(formResponses[SHORT_NAME_MAP.PREV_APPLICATION_TYPE]);
+
+  const noPrevApp =
+    formResponses[SHORT_NAME_MAP.PREV_APPLICATION] ===
+    RESPONSES.PREV_APPLICATION_2;
+
+  const preAppDateBefore = [
+    RESPONSES.PREV_APPLICATION_YEAR_1A,
+    RESPONSES.PREV_APPLICATION_YEAR_1B,
+    RESPONSES.PREV_APPLICATION_YEAR_1C,
+  ].includes(formResponses[SHORT_NAME_MAP.PREV_APPLICATION_YEAR]);
+
+  const courtMartial =
+    formResponses[SHORT_NAME_MAP.COURT_MARTIAL] === RESPONSES.COURT_MARTIAL_1;
+
+  const transgender =
+    formResponses[SHORT_NAME_MAP.REASON] === RESPONSES.REASON_5;
+  const intention =
+    formResponses[SHORT_NAME_MAP.INTENTION] === RESPONSES.INTENTION_1;
+  const dischargeYear = formResponses[SHORT_NAME_MAP.DISCHARGE_YEAR];
+  const dischargeMonth = formResponses[SHORT_NAME_MAP.DISCHARGE_MONTH] || 1;
+
+  const oldDischarge =
+    moment().diff(moment([dischargeYear, dischargeMonth]), 'years', true) >= 15;
```

Comment:
Consider: we should avoid using the `moment` package as it is deprecated. I wouldn't think Platform would want to eliminate its usage in the whole VW repo any time soon, but maybe it could improve performance for our app specifically if we use `date-fns` instead? That package is already in the repo and in use.